cordova-common/spec/CordovaCheck.spec.js

```diff
@@ -30,11 +30,12 @@ describe('findProjectRoot method', function() {
         process.env.PWD = origPWD;
         process.chdir(cwd);
     });
+function removeDir(someDirectory) {
+    shell.rm('-rf', someDirectory);
+}
     it('should return false if it hits the home directory', function() {
         var somedir = path.join(home, 'somedir');
-        this.after(function() {
```

Comment:
i like that you factored this out into its own function. What did `this.after` do before? Doesn't look like your function exactly replicates `this.after`, but not sure if this.after was ever needed here.